adapters/repos/db/index.go

```diff
@@ -441,7 +453,9 @@ func (i *Index) addProperty(ctx context.Context, props ...*models.Property) erro
 	eg := enterrors.NewErrorGroupWrapper(i.logger)
 	eg.SetLimit(_NUMCPU)
 
-	i.ForEachShard(func(key string, shard ShardLike) error {
+	// Ensure we only iterate on loaded shards as lazy loaded ones will ensure properties from the class have their
+	// bucket initiated on startup.
+	i.ForEachLoadedShard(func(key string, shard ShardLike) error {
```

Comment:
This change is not strictly necessary but I assumed that the effect is similar to the one below. Let me know what you think. If we want to reduce the blast impact of the change removing this one would ensure we don't touch anything else.